roles/openshift_clock_facts/tasks/main.yaml

```diff
@@ -0,0 +1,41 @@
+---
+- fail:
+    msg: "openshift_clock_method must be set to either 'none', 'auto', 'ntpd', or 'chrony'"
+  when:  openshift_clock_method is defined and
+    openshift_clock_method not in ["none", "auto", "ntpd", "chrony"]
```

Comment:
picking nits here, but instead of using fail/when here, these could be validated in the openshift_facts module. See the validate_local_facts method: https://github.com/openshift/openshift-ansible/blob/135f37c77145b08521a921dca81b843591b82c29/roles/openshift_facts/library/openshift_facts.py#L1595